peak_core/peak_core.py

```diff
@@ -37,10 +37,11 @@ class _PeakWrapper(metaclass=_PeakWrapperMeta):
     def __init__(self, peak_generator):
         pe = peak_generator(hwtypes.BitVector.get_family())
         assert issubclass(pe, peak.Peak)
-        pe = pe.__call__
-        (self.__instr_name, self.__instr_type) = pe._peak_isa_
-        self.__inputs = pe._peak_inputs_
-        self.__outputs = pe._peak_outputs_
+        self.__instr_name = 'inst'
```

Comment:
is doing this by name robust? is that part of the spec of peak? it's ok either way, should maybe just add a comment describing that.

Reply:
the name 'inst' is specific to Lassen and is not specific to a generic Peak program. I guess Lassen should hardcode this in it's codebase so that garnet could use it.